tests/lib/rules/padded-blocks.js

```diff
@@ -227,6 +265,17 @@ ruleTester.run("padded-blocks", rule, {
                     column: 1
                 }
             ]
+        },
+        {
+            code: "switch (a) {\n\ncase 0: foo();\n}",
+            options: [{"switches": "never"}],
+            errors: [
+                {
+                    message: NEVER_MESSAGE,
+                    line: 1,
+                    column: 1
```

Comment:
I'd also like to see an invalid `"never"` test (either a second error in this one or a second test) with extra space at the end of the `switch` body to make sure it gets the correct end location.